cli/fmt.rs

```diff
@@ -234,21 +227,41 @@ fn is_supported(path: &Path) -> bool {
 
 pub fn collect_files(
   files: Vec<PathBuf>,
+  ignore: Vec<PathBuf>,
 ) -> Result<Vec<PathBuf>, std::io::Error> {
   let mut target_files: Vec<PathBuf> = vec![];
+  // FIXME the unwrap here panicks when we add a ignore file/folder which doesn't exist
```

Comment:
Regression? Should be addressed here then.

Reply:
Non-existent paths passed to `ignore` should be skipped